### PR TITLE
Restore default hotplug attachment pod request to original values

### PR DESCRIPTION
**Pull request by awels**
**What this PR does / why we need it**:
As a quick fix we modified the requests and limit ratio to be 1. However this is not ideal since it reserves a lot of cpu and memory for that container that doesn't really do anything at all. Its purpose is to attach a volume to the node.

We added the ability to set the requests and limits for all the supporting containers in (#9422). At this point anyone can set the ratio to what they want. So lets restore the defaults to the original values, so that people that don't care don't end up using a lot of memory for no reason.

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #

**Special notes for your reviewer**:

**Checklist**

This checklist is not enforcing, but it's a reminder of items that could be relevant to every PR.
Approvers are expected to review this list.

- [ ] Design: A [design document](https://github.com/kubevirt/community/tree/main/design-proposals) was considered and is present (link) or not required
- [ ] PR: The PR description is expressive enough and will help future contributors
- [ ] Code: [Write code that humans can understand](https://en.wikiquote.org/wiki/Martin_Fowler#code-for-humans) and [Keep it simple](https://en.wikipedia.org/wiki/KISS_principle)
- [ ] Refactor: You have [left the code cleaner than you found it (Boy Scout Rule)](https://learning.oreilly.com/library/view/97-things-every/9780596809515/ch08.html)
- [ ] Upgrade: Impact of this change on upgrade flows was considered and addressed if required
- [ ] Testing: New code requires [new unit tests](https://github.com/kubevirt/kubevirt/blob/main/docs/reviewer-guide.md#when-is-a-pr-good-enough). New features and bug fixes require at least on e2e test
- [ ] Documentation: A [user-guide update](https://github.com/kubevirt/user-guide/) was considered and is present (link) or not required. You want a user-guide update if it's a user facing feature / API change.
- [ ] Community: Announcement to [kubevirt-dev](https://groups.google.com/g/kubevirt-dev/) was considered

**Release note**:

```release-note
Restored hotplug attachment pod request/limit to original value
```

### CODE REVIEW

**Review by xpivarc — COMMENTED** [review]
/approve

File: pkg/virt-controller/services/template_test.go [hunk 1]
```diff
@@ -4070,7 +4070,7 @@ var _ = Describe("Template", func() {
 			Expect(memLimit).To(Equal(expMemLimitQ.Value()))
 			expCpuReqQ := resource.MustParse("100m")
 			Expect(cpuReq).To(Equal(expCpuReqQ.Value()))
-			expMemReqQ := resource.MustParse("80M")
+			expMemReqQ := resource.MustParse("2M")
```
Comment (fossedihelm) [inline pkg/virt-controller/services/template_test.go:4073]: I'd expect `expCpuReqQ := resource.MustParse("100m")` to be `expCpuReqQ := resource.MustParse("10m")`

Comment (fossedihelm) [reply pkg/virt-controller/services/template_test.go:4073]: Sorry, reading my own comment I realize this is too criptic.
What I mean is that here the expected memory request was reduced back to `2M`.
I would expect the same to happen for the cpu request too, IOW to be rolled back to `10m`, but it is not.
Can you explain? Thanks

Comment (awels) [reply pkg/virt-controller/services/template_test.go:4073]: No you are right, have to check why just the one, and not the other.

Comment (awels) [reply pkg/virt-controller/services/template_test.go:4073]: So turns out the unscaled value of 10m and 100m is both '1' and thus matches, instead I need to use MilliValue, and that fixes the issue.

**Comment by fossedihelm** [comment]
Thank you! 
/lgtm
/retest-required

**Pull request merged**